hack/kind/kind.sh

```diff
@@ -14,7 +14,7 @@ function check() {
 		echo "'kind' is installed. '$(kind --version)' will be used"
 		return 0
 	else
-		echo "'kind' executable is not avialable in PATH"
+		echo "'Kind' executable is not available in PATH"
```

Comment:
Why ?

Reply:
It's just a typo error :sweat_smile:

Reply:
why typo ? the binary is `kind` not `Kind`